src/engine/Algebra.ts

```diff
@@ -1,10 +1,13 @@
+/// <reference path="Interfaces/ConstructorBags/AlgebraOptions.ts" />
+
 module ex {
 
    /**
     * A 2D vector on a plane. 
     */
-   export class Vector {
-
+   export class Vector implements IVectorOptions {
```

Comment:
Sure, but we would like to take advantage of the compiler errors/warnings that would be generated when adding new options. Removing the implementing would make it harder to ensure every class correctly implements the properties we expect.